frame/staking/src/tests.rs

```diff
@@ -18,7 +18,7 @@
 //! Tests for the module.
 
 use super::{ConfigOp, Event, *};
-use frame_election_provider_support::{ElectionProvider, SortedListProvider, Support};
+use frame_election_provider_support::{ElectionProvider, ReadOnlySortedListProvider, Support};
```

Comment:
I've mentioned this before as well: we should have tests in the staking side that demonstrate that, regardless of who's listening on the other side, staking is calling the right events into `type EventListener`.